runtests.py

```diff
@@ -8,18 +8,20 @@
 File for running tests programmatically.
 """
 
+# Standard library imports
+import sys
+
 # Third party imports
-import qtpy
 import pytest
 
 
```

Comment:
That does not work. But it suggested to me that we should intervene in before the `import pytest` in `runtests.py` so I'm trying that now.

Interesting that you are thinking of running `runtests.py` locally; I find it print too much cruft to be useful.